packages/opencensus-core/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "allowUnreachableCode": true,
+    "noUnusedParameters": false,
+    "noImplicitAny": false,
+    "noImplicitThis": false,
+    "noEmitOnError": false,
+    "pretty": true,
+    "strict": false,
+    "module": "commonjs",
+    "target": "es6",
+    "sourceMap": false
+  },
+  "include": [
+    "src/*.ts",
```

Comment:
This needs to be `"src/**/*.ts"` or `npm run fix` will not work.

Reply:
Done!